Misc/ACKS

```diff
@@ -2094,5 +2094,6 @@ Jelle Zijlstra
 Gennadiy Zlobin
 Doug Zongker
 Peter Åstrand
+Vlad Efanov
```

Comment:
I think they want to keep this roughly in alphabetical order by last name.

Reply:
I'm sorry. Did not pay attention on comment.